src/dialects/mssql/schema/columncompiler.js

```diff
@@ -62,7 +62,9 @@ assign(ColumnCompiler_MSSQL.prototype, {
 
   datetime: 'datetime',
 
-  timestamp: 'datetime',
+  timestamp(useTz) {
```

Comment:
use options object instead of single boolean parameter.

Reply:
Regarding the object parameter on the function, it will be different between the dialects